nixos/modules/services/misc/libreddit.nix

```
@@ -56,6 +56,26 @@ in
           AmbientCapabilities = lib.mkIf (cfg.port < 1024) [ "CAP_NET_BIND_SERVICE" ];
           Restart = "on-failure";
           RestartSec = "2s";
+          DeviceAllow = "";
+          LockPersonality = "yes";
```

```suggestion
          LockPersonality = true;
```

Not 100% sure about this, but I usually rather see `true` rather than `"yes"` in nixpkgs systemd options.

---

also change that for all other options